### DISCUSSION TITLE
Purpose of bootstrap server as Load Balancer (with external IP)

### DISCUSSION THREAD

**Discussion by trietsch** [strimzi/strimzi-kafka-operator#4430] [Q&A] [answered]
What's the purpose of the bootstrap server, when using a Load Balancer with an external IP address? Passing 1, 2, or even more broker addresses that are reachable through Load Balancers should also do the job, right?
The external IP address used for the bootstrap server is yet another IP address that is paid for in public clouds, so I'm trying to understand it's benefit over just using one or multiple broker addresses.

**Comment by trietsch** [comment]
Relates to this question, but the question above is more focused on the purpose of the bootstrap server for Strimzi (with Load Balancer as external interface): https://github.com/strimzi/strimzi-kafka-operator/discussions/4364

**Comment by scholzj** [comment, marked as answer]
It is a convenience thing which makes it easier to configure the clients => you configure just one address and not a lot of them. I agree that it might not be absolutely necessary.

**Reply by trietsch** [reply]
Thanks for your quick reply! Is it correct that there currently is no way of disabling the bootstrap address in a Load Balancer setting?

**Reply by scholzj** [reply]
Yes, that is correct. But if you want to open a PR to add some option to disable it feel free. You can also open an enhancement issue and see if someone gets to it.

**Reply by trietsch** [reply]
Alright, I'll open an issue for this. I might be able to create a PR. Thanks!

**Reply by trietsch** [reply]
Reference to the issue: https://github.com/strimzi/strimzi-kafka-operator/issues/4433
